add docs (#347)
elasticio/api-docs: source/includes/_recipes.md

                "subject": "pets[0].id",
                "to": "pets[0].name"
              },
              "mapper_type": "jsonata"
            },
            "source": "step_1",
            "target": "step_2"
          }
        ],
        "nodes": [
          {
            "name": "Step name",
            "description": "Step description",
            "command": "elasticio/petstore:getPetsByStatusWithGenerators@latest",
            "id": "step_1"
          },
          {
            "name": "Step name",
            "description": "Step description",
            "command": "elasticio/email:send@latest",
            "fields": {
              "dontThrowErrorFlg": true
            },
            "id": "step_2"
          }
        ]
      }
    },
    "relationships": {
      "user": {
        "data": {
          "id": "{USER_ID}",
          "type": "user"
        },
        "links": {
          "self": "/v2/users/{USER_ID}"
        }
      },
      "workspace": {
        "data": {
          "id": "{WORKSPACE_ID}",
          "type": "workspace"
        },
        "links": {
          "self": "/v2/workspaces/{WORKSPACE_ID}"
        }
      }
    }
  },
  "meta": {
    
  }
}
```

> This resource allows you to create a new recipe without required trigger/action fields.

Reviewer: ```suggestion
This resource allows you to create a new recipe.
```

              },
              "mapper_type": "jsonata"
            },
            "source": "step_1",
            "target": "step_2"
          }
        ],
        "nodes": [
          {
            "name": "Step name",
            "description": "Node description",
            "command": "elasticio/petstore:getPetsByStatusWithGenerators@latest",
            "fields": {
              "status": "pending"
            },
            "id": "step_1"
          },
          {
            "name": "Step name",
            "description": "Node description",
            "command": "elasticio/email:send@latest",
            "fields": {
              "dontThrowErrorFlg": true
            },
            "id": "step_2"
          }
        ]
      }
    },
    "relationships": {
      "user": {
        "data": {
              "id": "{USER_ID}",
          "type": "user"
        },
        "links": {
          "self": "/v2/users/{USER_ID}"
        }
      },
      "workspace": {
        "data": {
          "id": "{WORKSPACE_ID}",
          "type": "workspace"
        },
        "links": {
          "self": "/v2/workspaces/{WORKSPACE_ID}"
        }
      }
    }
  },
  "meta": {
    
  }
}
```

> This resource allows you to create a new recipe with required trigger/action fields.

Reviewer: ```suggestion
This resource allows you to create a new recipe.
```

        "edges": [
          {
            "config": {
              "mapper_type": "jsonata",
              "mapper": {
                "to": "pets[0].name",
                "subject": "pets[0].id",
                "textBody": "pets[0].status"
              },
              "condition": null
            },
            "source": "step_1",
            "target": "step_2"
          }
        ],
        "nodes": [
          {
            "command": "elasticio/petstore:getPetsByStatusWithGenerators@latest",
            "id": "step_1"
          },
          {
            "command": "elasticio/email:send@latest",
            "fields": {
              "dontThrowErrorFlg": true
            },
            "id": "step_2"
          }
        ]
      }
    },
    "relationships": {
      "user": {
        "data": {
          "id": "{USER_ID}",
          "type": "user"
        },
        "links": {
          "self": "/v2/users/{USER_ID}"
        }
      },
      "workspace": {
        "data": {
          "id": "{WORKSPACE_ID}",
          "type": "workspace"
        },
        "links": {
          "self": "/v2/workspaces/{WORKSPACE_ID}"
        }
      }
    }
  },
  "meta": {
    
  }
}
```

> This resource allows you to retrieve a recipe by its identifier. If the recipe with given ID does not belong to the current

Reviewer: I'd add the section

```
### Authorization

This request is authorized to a member of the corresponding contract only.
```

Instead of the sentence ```If the recipe with given ID does not belong to the current```

> This resource allows you to retrieve a recipe by its identifier. If the recipe with given ID does not belong to the current

Reviewer: `identifier` -> `ID`

        "links": {
          "self": "/v2/workspaces/{WORKSPACE_ID}"
        }
      }
    }
  },
  "meta": {
    
  }
}
```

This resource allows you to update the given recipe.

### HTTP Request

`PATCH {{ api_base_url }}/v2/recipes/{RECIPE_ID}`

### URL Parameters

| Parameter | Required | Description |
| :-------- | :------- | :---------- |
| RECIPE_ID | yes      | Recipe ID   |

### Body Parameters

| Parameter                         | Required | Description                                                                     |
| :-------------------------------- | :------- | :------------------------------------------------------------------------------ |
| type                              | yes      | A value must be `recipe`                                                        |
| attributes.name                   | yes      | Recipe name                                                                     |
| attributes.cron                   | no       | Cron expression                                                                 |
| attributes.graph                  | yes      | Recipe graph representing component connections                                 |

> | relationships.workspace.data.id   | yes      | An Id of the Workspace                                                          |

Reviewer: ```suggestion
| relationships.workspace.data.id   | yes      | MUST be the same as the RECIPE_ID                                                         |
```

This request is authorized for a user with the `workspaces.recipe.edit` permission.

### Returns

Returns the updated recipe

## Activate a recipe

> Example request

```shell
curl {{ api_base_url }}/v2/recipes/{RECIPE_ID}/activate \
   -X POST \
   -u {EMAIL}:{APIKEY} \
   -H 'Accept: application/json' \
   -H 'Content-Type: application/json' -d '
   {
     "data": {
       "type": "recipe-activation-config",
       "attributes": {
         "name": "Flow, created from Recipe",
         "credentials": {
           "step_1": "{STEP_ID}"
         },
         "variables": {
           "TO_EMAIL": "goose@elastic.io",
           "NAME_IN_SUBJECT": "Neochen Jubata"
         }
       },
       "relationships": {
         "workspace": {
           "data": {
             "type": "workspace",
             "id": "{WORKSPACE_ID}"
           }
         }
       }
     }
   }'
```

> Example response

```http
HTTP/1.1 201 Created
{
  "data": {
    "relationships": {
      "flow": {
        "data": {
          "type": "flow",
          "id": "{FLOW_ID}"
        },
        "links": {
          "self": "/v2/flows/{FLOW_ID}"
        }
      }
    }
  },
  "meta": {
    
  }
}
```

> Create a flow from a recipe.

Reviewer: ```suggestion
Create a flow from a recipe. If the recipe contains a component, which requires a credential, it must be provided among the request payload.
```
